games/x.yaml

```
@@ -796,6 +796,21 @@
   - https://user-images.githubusercontent.com/8344157/131311065-b77cb24f-9b10-4265-a61f-69d304040da0.png
   - https://user-images.githubusercontent.com/8344157/117028249-940d9e80-ad0e-11eb-84da-70fdbe41e0f2.png
 
+- name: xmine
+  originals:
+  - Minesweeper
+  type: clone
+  repo: 'https://ftp.openbsd.org/pub/OpenBSD/distfiles/#:~:text=xmine'
+  url: 'https://happypenguin.altervista.org/sheet.php?gameid=41'
```

```suggestion
  url: https://www.mobygames.com/game/151978/xmine/
```
this has a much better description of the game